app/services/doc_auth/mock/result_response.rb

```diff
@@ -105,10 +110,14 @@ def self.create_network_error_response
       end
 
       def doc_auth_success?
-        (doc_auth_result_from_uploaded_file == 'Passed' ||
+        return false unless id_type_supported?
+        return false if transaction_status_from_uploaded_file == 'failed'
+        return true if transaction_status_from_uploaded_file == 'passed'
+        return false if doc_auth_result_from_uploaded_file == 'Failed'
```

Comment:
Is this line in case there are `yml` files out there that haven't implemented `transaction_status`?

If so, it does seem like it could be worthwhile enforcing folks to include that, rather than keeping this method for old code.

Reply:
good point .. currently we do not force folks to include doc_auth_result ... so replicating this behavior with transaction_status